### DISCUSSION TITLE
Why is the lidar status in 16 dimensions and what do they represent?

### DISCUSSION THREAD

**Discussion by needtorch** [PKU-Alignment/safety-gymnasium#122] [Q&A]
I use the environment "Safety[Agent]Goal1-v0". After the step, I found that next_observation contains the data returned by the radar. I am confused about the 16-dimensional data returned by the goal lidar and the 16-dimensional data returned by the hazards lidar. What do they represent?

**Comment by muchvo** [comment]
Please refer to [this page](https://safety-gymnasium.readthedocs.io/en/latest/components_of_environments/objects.html#pseudo-lidar), which provides a detailed description of the radar mechanism.
